concepticondata/conceptlists/Lionnet-1985-364.tsv

```diff
@@ -0,0 +1,365 @@
+Lionnet-1985-364-157	157	gato montés	wild cat	2241	WILD CAT
+Lionnet-1985-364-158	158	grande	big	1202	BIG
+Lionnet-1985-364-159	159	granizo	hail	609	HAIL
+Lionnet-1985-364-160	160	grasa	grease	323	FAT (ORGANIC SUBSTANCE)
+Lionnet-1985-364-161	161	gritar	shout	715	SHOUT
+Lionnet-1985-364-162	162	gritar, cantar	sing, shout	1261	SING
+Lionnet-1985-364-163	163	grulla	crane (bird)	1830	CRANE
+Lionnet-1985-364-164	164	guacamaya	parrot	882	PARROT
+Lionnet-1985-364-165	165	guajolote	turkey	2386	TURKEY
+Lionnet-1985-364-166	166	guardar	keep, protect	2900	PROTECT
+Lionnet-1985-364-167	167	gusano	worm	1219	WORM
+Lionnet-1985-364-168	168	haber luz	be light		
+Lionnet-1985-364-169	169	hablar	talk	2708	TALK
+Lionnet-1985-364-170	170	hacer calor	be warm		
+Lionnet-1985-364-171	171	hallar	find	719	FIND
+Lionnet-1985-364-172	172	hartarse	become fed up		
+Lionnet-1985-364-173	173	heder	stink	1587	SMELL (STINK)
+Lionnet-1985-364-174	174	henderse	cleave		
+Lionnet-1985-364-175	175	hermana mayor	older sister	1758	OLDER SISTER
+Lionnet-1985-364-176	176	hermana mayor del padre	father's older sister	3041	PATERNAL AUNT (FATHER'S OLDER SISTER)
+Lionnet-1985-364-177	177	hermana menor	younger sister	1761	YOUNGER SISTER
+Lionnet-1985-364-178	178	hermana menor de la madre	mother's younger sister	3043	MATERNAL AUNT (MOTHER'S YOUNGER SISTER)
+Lionnet-1985-364-179	179	hermana menor del padre	father's younger sister		
+Lionnet-1985-364-180	180	hermano mayor	older brother	1759	OLDER BROTHER
+Lionnet-1985-364-181	181	hermano mayor del padre	father's older brother	2173	PATERNAL UNCLE (FATHER'S OLDER BROTHER)
+Lionnet-1985-364-182	182	hermano menor	younger brother	1760	YOUNGER BROTHER
+Lionnet-1985-364-183	183	hermano menor de la madre	mother's younger brother	3250	MATERNAL UNCLE (MOTHER'S YOUNGER BROTHER)
+Lionnet-1985-364-184	184	hervirse	boil (in water)		
+Lionnet-1985-364-185	185	hígado	liver	1224	LIVER
+Lionnet-1985-364-186	186	higo	fig	322	FIG
+Lionnet-1985-364-187	187	hija del padre	daughter of father	3778	DAUGHTER (OF MALE EGO)
+Lionnet-1985-364-188	188	hijo	son	1620	SON
+Lionnet-1985-364-189	189	hijo del padre	son of father	3779	SON (OF MALE EGO)
+Lionnet-1985-364-190	190	hilar	spin	121	SPIN
+Lionnet-1985-364-191	191	hoja	leaf	628	LEAF
+Lionnet-1985-364-192	192	hombre	man	1554	MAN
+Lionnet-1985-364-193	193	huaraches	sandals		
+Lionnet-1985-364-194	194	hueso	bone	1394	BONE
+Lionnet-1985-364-195	195	huevo	egg	744	EGG
+Lionnet-1985-364-196	196	humear	to smoke (fire)	2859	SMOKE (EMIT SMOKE)
+Lionnet-1985-364-197	197	humo	smoke	778	SMOKE (EXHAUST)
+Lionnet-1985-364-198	198	invierno	winter	1471	WINTER
+Lionnet-1985-364-199	199	ir	go	695	GO
+Lionnet-1985-364-200	200	jilotear	for an ear of corn to form		
+Lionnet-1985-364-201	201	largo	long	1203	LONG
+Lionnet-1985-364-202	202	lavar	wash	1453	WASH
+Lionnet-1985-364-203	203	lejos	far	1406	FAR
+Lionnet-1985-364-204	204	lengua	tongue	1205	TONGUE
+Lionnet-1985-364-205	205	león	lion (cougar)	1386	LION
```

Comment:
Given that there are no lions in the Americas (and the English translation indicates COUGAR), this should probably be unmapped